.github/workflows/build.yml

```diff
@@ -7,9 +7,6 @@ on:
     pull_request: ~
     release:
         types: [created]
-    schedule:
```

Comment:
Removed this because it has this unintended consequence:

<img width="1305" alt="image" src="https://github.com/Sylius/SyliusAdminApiBundle/assets/2412177/c5d9965d-bce8-44e7-9d09-590e6d1d931b">